src/dev/impl/DevToys/ViewModels/Tools/Generators/HashGenerator/HashGeneratorToolViewModel.cs

```diff
@@ -164,12 +199,23 @@ private async Task<string> CalculateHashAsync(string alrogithmName, string text)
 
             try
             {
-                var algorithmProvider = HashAlgorithmProvider.OpenAlgorithm(alrogithmName);
-
-                IBuffer buffer = CryptographicBuffer.ConvertStringToBinary(text, BinaryStringEncoding.Utf8);
-                buffer = algorithmProvider.HashData(buffer);
-
                 string? hash = "";
+                IBuffer? buffer = null;
```

Comment:
Perhaps we could add some unit tests to https://github.com/veler/DevToys/blob/main/src/tests/DevToys.Tests/Providers/Tools/HashGeneratorTests.cs